data/basics/testing/userBalance.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2019-2022 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package testing
+
+import (
+	"github.com/algorand/go-algorand/data/basics"
+)
+
+// MakeAccountData returns a AccountData with non-empty voting fields for online accounts
+func MakeAccountData(status basics.Status, algos basics.MicroAlgos) basics.AccountData {
```

Comment:
Do we know why this function used to live outside of a test file? You mentioned in the description it's only used for tests, so it makes sense to pull it into a test file, but I'm just curious if it indicates a larger refactor is needed around this.